### PR TITLE
 fix: standardize output table name in training and prediction models 

### PR DESCRIPTION
**Pull request by shekhar-rudder**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/predictions/profiles_mlcorelib/py_native/warehouse.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def standardize_ref_name(warehouse_type: str, ref_name: str):
+    # This is the equivalent of the StandardizeRefName function in wht.
+    # It is not being used by most of pynative models since they use "warehouse.CreateReplaceTableAs"
+    # which has this logic built in.
+    if warehouse_type == "snowflake":
+        return ref_name.upper()
+    if warehouse_type == "redshift":
+        ref_name.lower()
```
Comment (dpatchigolla) [inline src/predictions/profiles_mlcorelib/py_native/warehouse.py:8]: This does not modify the string inplace. Can you add return statement here? Or assign it back to ref_name?

File: tests/integration/utils.py [hunk 2]
```diff
@@ -23,7 +23,10 @@ def get_pynative_output_folder():
         return None
     items = os.listdir(seq_no_dir)
     directories = [
-        item for item in items if os.path.isdir(os.path.join(seq_no_dir, item))
+        # This logic will fail if there are multiple sequence numbers
```
Comment (dpatchigolla) [inline tests/integration/utils.py:26]: We will have multiple seq numbers often right?

Comment (shekhar-rudder) [reply tests/integration/utils.py:26]: Locally yes but not in the CI pipeline

File: src/predictions/profiles_mlcorelib/py_native/warehouse.py (hunk 1 above)
Comment (shekhar-rudder) [reply src/predictions/profiles_mlcorelib/py_native/warehouse.py]: Added return statement

**Review by dpatchigolla — APPROVED** [review]

**Pull request merged**